### PR TITLE
closes #190

### LINKED ISSUE #190: fix menu closing

### PR DESCRIPTION
**Pull request by cmperez**

### CODE REVIEW

**Comment by owen-d** [comment]
:+1:

File: client/js/components/menu.jsx [hunk 1]
```diff
@@ -4,9 +4,18 @@ var $ = require("jquery");
 var Menu = React.createClass({
   componentDidMount: function() {
     var DROP_DOWN_SPEED = 300;
+    var TOGGLED = false;
```
Comment (dduugg) [inline client/js/components/menu.jsx]: all caps is generally reserved for constants. since you're reassigning the var, camel case is recommended.

**Comment by cmperez** [comment]
good catch, dug. fixed

File: client/js/components/menu.jsx [hunk 2]
```diff
@@ -4,9 +4,18 @@ var $ = require("jquery");
...
     var DROP_DOWN_SPEED = 300;
+    var isToggled = false;
     $(document).ready(function() {
       $(".menu").click(function () {
         $("#dropdown").slideToggle(DROP_DOWN_SPEED);
+        isToggled = !isToggled;
+      });
+
+      $(document).on('keydown', function(event) {
+        if (event.keyCode === 27 && isToggled) {
```
Comment (dduugg) [inline client/js/components/menu.jsx]: not a UI guy but a comment here explaining what `27` is would be helpful

**Pull request merged**